master/internal/api_trials.go

```diff
@@ -29,12 +33,16 @@ import (
 )
 
 const (
-	batchSize = 1000
+	batchSize                     = 1000
```

Comment:
If we have two different batch sizes, should be change the name of this one to be more specific? Or is that part of the larger refactor follow-on?